frontend/src/types.ts

```diff
@@ -192,6 +194,11 @@ interface BasePropertyFilter {
     label?: string
 }
 
+export interface EventProperty {
```

Comment:
There's already `SelectOption` which is the exact same interface. Is this used for something similar? We could use the same interface.